cellprofiler/modules/__init__.py

```diff
@@ -108,6 +108,7 @@
                         'straightenworms': 'StraightenWorms',
                         'matchtemplate': 'MatchTemplate',
                         'trackobjects': 'TrackObjects',
+                        'tram': 'TrAM',
```

Comment:
Makes sense. Can we decide on a name now, since we need to lock it down in the chapter we're writing. How about TrackQuality, TrackMetrics, TrackQC? Other suggestions?